sdks/python/apache_beam/transforms/external.py

```diff
@@ -224,6 +224,44 @@ def _normalize(coder_proto):
             for tag, pcoll in self._expanded_transform.outputs.items()})
 
 
+class JavaJarExpansionService(object):
+  """An expansion service based on an Java Jar file.
```

Comment:
Should this live in `apache_beam.runners.portability.expansion_service`?  Or should we find some common home for both `JavaJarExpansionService` and `ExpansionServiceServicer`

Reply:
Well, it's not really part of runners. (Also, I hope eventually the "portability" part becomes redundant :).)

Reply:
Sorry, what I was trying to say was that we already have one expansion service class, `apache_beam.runners.portability.expansion_service.ExpansionServiceServicer`, so it's unclear to me why `JavaJarExpansionService` is unique enough from `ExpansionServiceServicer` to warrant living in a different module.  Naively it seems like it would make more sense for them to live together.

Reply:
Ah, I see what you mean now. 

apache_beam.runners.portability.expansion_service.ExpansionServiceServicer is for serving Python transforms to other languages (i.e. calling from another language to Python). On the other hand, JavaJarExpansionService and and friends are about calling from Python to another language. I think it makes the most sense for the latter to be defined close to the ExternalTransform class with which it is used.  (It is a convenience that the server and client objects in Python share the same API, so you can use the Server object in the place of a Client for testing.)